governance/polls/Ratification Poll for Growth Core Unit Budget, GRO-001 (MIP40c3-SP37) - October 11, 2021.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Ratification Poll for Growth Core Unit Budget, GRO-001 (MIP40c3-SP37) - October 11, 2021
+summary: MIP40c3-SP37 adds the budget for Core Unit GRO-001: Growth for November 2021 through June 2022.
```

Comment:
You need to remove the colon after "GRO-001" as this is breaking the formatting of the preamble